driver/src/main/scala/org/mongodb/scala/model/PushOptions.scala

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright 2015 MongoDB, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.mongodb.scala.model
-
-import com.mongodb.client.model.{ PushOptions => JPushOptions }
-
-/**
- * The options to apply to a `\$push` update operator.
- *
- * @since 1.0
- */
-object PushOptions {
```

Comment:
Because it wasn't needed as a separate file - so was folded into the package like the other simple objects.